eo-parser/src/main/java/org/eolang/parser/Syntax.java

```diff
@@ -178,4 +180,77 @@ private List<Text> lines() {
         return new ListOf<>(new Split(new TextOf(this.input), "\r?\n"));
     }
 
+    /**
+     * Accumulates all parsing errors.
+     *
+     * @since 0.30.0
+     */
+    private static final class ParsingErrors extends BaseErrorListener
+        implements ANTLRErrorListener, Iterable<Directive> {
```

Comment:
@yegor256 `BaseErrorListener` already implements the `ANTLRErrorListener`. Do we need to specify this twice?

Reply:
@volodya-lombrozo I believe, it's a good practice to mention both, the interface and the base class, here is why. If we only mention the base class, future developers may get confused which interface in particular we implement, when/if they decide to replace the base class with something different or simply reimplement my `ParsingErrors` from scratch, without the base class.